README.md

```diff
@@ -218,3 +218,17 @@ Starting with version 2.8, there is the `ide` flavor of the images, which comes
 Store your preferred password in this variable if you need to password protect the IDE environment.
 
 [See docs](https://docs.docksal.io/tools/ide/) for instructions on using Coder in Docksal.
+
+## Composer
+
+Versions 1 and 2 are a part of the container. While not all projects will work with version 2 quite yet 
+there is the ability to role back and use version 1 by setting the `COMPOSER_DEFAULT_VERSION` variable to `1`.
```

Comment:
Hmm, there should be some rework of the sentence about rolling back. First, if version 2 is set as the default, it should say that to may clear why you would "roll back." I wouldn't use that term though. Something like...

Versions 1 and 2 are both installed in the container. Version 2 is set as the default version, but while not all projects may be able to work with version 2 quite yet, version 1 is available by setting the `COMPOSER_DEFAULT_VERSION` variable to `1`.